### PR TITLE
simple interest 

### PR DESCRIPTION
**Pull request by Anjali-1801**
My first Contribution.

### CODE REVIEW

File: simple_interest.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+int main(void)
+{
+    int p,n,A;
+    float r,si;
+    printf("Enter p:Rs.");
+    scanf("%d",&p);
+    printf("Enter n:");
+    scanf("%d",&n);
+    printf("Enter r:");
+    scanf("%f",&r);
```
Comment (sarvesh-0) [inline simple_interest.c:14]: Could you please use only a single pritf() and scanf() function for p, n and r

Comment (Anjali-1801) [reply simple_interest.c]: Sir, I made the required changes and pre-assigned the values for p, n, and r.

**Review by sarvesh-0 — APPROVED** [review]

File: simple_interest.c [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+and the rate is 10%. Also, calculate the total amount at the end of 1 year*/
+
+#include <stdio.h>
+int main(void)
+{
+    int p,n,A;
+    float r,si;
+    printf("Enter p, n, r: ");
+    scanf ("%d %d %f", &p, &n, &r);
+    printf("Simple Interest (si): ");
```
Comment (sarvesh-0) [inline simple_interest.c:11]: write the same logic in just one or two line.

Comment (sarvesh-0) [reply simple_interest.c:11]: I will myself do it,

**Pull request merged**